Makefile

```diff
@@ -172,8 +172,7 @@ cppcheck-long: FORCE
 
 ## pep8        : check Python code style
 pep8: $(PYSOURCES) $(wildcard tests/*.py)
-	pep8 --exclude=_version.py  --show-source setup.py khmer/ \
-		scripts/ tests/ oxli/ || true
+	pep8 --ignore E309,E226 setup.py khmer/*.py scripts/*.py tests/*.py oxli/*.py
```

Comment:
This is automatically generated by `versioneer`, so probably a good idea do exclude it from pep8.